controller.js

```diff
@@ -7,4 +8,12 @@ function getAllTopics(req, res, next){
         next(err)
     }) 
 }
-module.exports = {getAllTopics}
\ No newline at end of file
+
+function getApi(req, res, next) {
+    getAllEndPoints().then(endpoints => {
+        res.status(200).json(endpoints);
```

Comment:
If this ends up being the behaviour you use on this endpoint, when we send it back lets send it on an object as normal with an appropriate key.